src/python/pants/backend/python/goals/setup_py.py

```diff
@@ -169,12 +167,6 @@ class ExportedTargetRequirements(DeduplicatedCollection[str]):
     sort_input = True
 
 
-@dataclass(frozen=True)
-class SetupPySourcesRequest:
```

Comment:
No longer needed - we request SetupPySources directly from a SetupPyChrootRequest.